commerce_coordinator/apps/titan/serializers.py

```diff
@@ -0,0 +1,26 @@
+""" Serializers for data manipulated by commerce-coordinator titan API endpoints. """
+
+from rest_framework import serializers
+
+
+class OrderFulfillSerializer(serializers.Serializer):
```

Comment:
Instead of having a create/update, what do you think about https://www.django-rest-framework.org/api-guide/serializers/#overriding-save-directly?

Reply:
Nice find! I prefer this since we don't need an `update()`. Thank you!

Reply:
Decided to remove the serializer assuming all of the data validation will be done on the `edx-platform` or `ecommerce` endpoint, and `commerce-coordinator` is just the messenger/carrier of data